examples/2d/2d_shapes.rs

```diff
@@ -34,11 +34,21 @@ fn setup(
         ..default()
     });
 
+    // Quad
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: meshes
+            .add(shape::Quad::new(Vec2::new(50., 100.)).into())
+            .into(),
+        material: materials.add(ColorMaterial::from(Color::BLUE)),
```

Comment:
Full-blue is pretty jarring next to the other colors which are a bit muted. Maybe something like `LIME_GREEN`?